### DISCUSSION TITLE
Profile Issues

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#36178] [Other Feature Feedback, Questions, & Ideas]
Hey everyone,

Does anyone know that if we contribute almost daily to GitHub then we get any goodies from them?

Also how can we receive any goodies from GitHub?

**Comment by FahdSeddik** [comment]
Hey @iLakshya 
No, you do not receive any rewards for being more active from GitHub.

**Comment by jfullstackdev** [comment]
GitHub Sponsors - make a useful app and others will sponsor it 

Open Source Projects - be consistent, be one of the assets of one Open Source project and they may pay you for that or one company will pay for you if you will customize the service fit for the business need of the company, if that is allowed by the open source project itself

GitHub As Your Skillset - I applied for several developer jobs in the Philippines, others are considering your small projects hosted on GitHub as your experience and I was hired because of one of my PHP / Laravel projects, using it to demo my skills, right here on GitHub, even if I was not an IT graduate
